calico_node/calico-bgp-daemon/main.go

```diff
@@ -45,13 +46,31 @@ const (
 	ETCD_CERT_FILE    = "ETCD_CERT_FILE"
```

Comment:
Any reason why these constants are exported? 

Also, names with underscores can be in CamelCase 

Ref: https://golang.org/doc/effective_go.html#mixed-caps